pkg/ebpf/c/tracee.bpf.c

```diff
@@ -6940,6 +6945,22 @@ struct {
     __type(value, u32);
 } net_cap_events SEC(".maps");
 
+// scratch area
+
+struct {
+    __uint(type, BPF_MAP_TYPE_PERCPU_ARRAY);
+    __uint(max_entries, 1);                    // simultaneous softirqs running per CPU (?)
```

Comment:
I'm not sure if a softirq, running right after a top half, that just preempted the bpf program, can be preempted by another softirq. For now, simply using another map for the network events scratch, was enough to avoid corruption.